### DISCUSSION TITLE
Page Break Issue in PDF output using BIRT.

### DISCUSSION THREAD

**Discussion by sharath492** [eclipse-birt/birt#1001] [Q&A] [answered]
Hello,

I am using BIRT Designer, I am having an issue of page break when generating the report as PDF.

After generating the PDF report, What happen is since the note from the input file is too long for the description column to accommodate some of the string is being pushed down to next line or being wrap to ensure that the note fits in the description column area which cause for the trailer details to be push to the next page.

I have tried many page break options like avoid before and after for tables/grid but nothing is working.

So Please give some suggestion to solve as soon as possible.

Please find the attachment of sample pdf for getting page break issue and rptdesign file and xml file.

Thanks & Regards,
Sharath
[stearn.pdf](https://github.com/eclipse/birt/files/8984419/stearn.pdf)
[Birt files.zip](https://github.com/eclipse/birt/files/8984422/Birt.files.zip)
.

**Comment by hvbtup** [comment]
I don't understand your report. It contains dozens of group footer rows which are empty except an empty label.
![grafik](https://user-images.githubusercontent.com/7644776/175963543-1e69eb55-c782-4515-965e-0253f8ccf100.png)

I guess these rows are causing the problem.
What is their purpose?

**Reply by sharath492** [reply]
Hello,

Even i'm also not sure but they are checking some conditions based on totalItems.

The issue is coming in the TrailerSecondNote.

**Comment by hvbtup** [comment, marked as answer]
I think there is a general misunderstanding here. If I understand correctly, you are using the "number of items" to calculate height and thus to hide some of the footer rows?
That is generally not recommended unless the height of all individual items and all headers and footers is constant.

In your case, the height needes for the notes is variable, thus this kind of height calculation doesn't make sense.

**Reply by sharath492** [reply]
Hi Hvbtup,

It has been solved now. I removed one empty label in between TrailerFirstNote and  TrailerSecondNote.
